CMakeLists.txt

```diff
@@ -58,6 +58,8 @@ if (NOT ${use_installed_dependencies})
             add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/deps/c-utility/testtools EXCLUDE_FROM_ALL)
         endif()
     endif()
+else()
+    find_package(azure_c_shared_utility REQUIRED config)
```

Comment:
>find_package(azure_c_shared_utility REQUIRED config) [](start = 4, length = 52)

Don't you need a find_package for umock-c

Reply:
^ same as this. Don't you need a find_package for all the dependencies listed under deps?